vector/v.in.lidar/v.in.lidar.html

```diff
@@ -148,8 +148,3 @@ <h2>AUTHOR</h2>
 (decimation, cats, areas, zrange)
 <br>
 based on <em>v.in.ogr</em> by various authors
-
-<!--
-<p>
-<i>Last changed: $Date$</i>
--->
```

Comment:
Line 76  unnecessary one empty line.

`<h2>REFERENCES</h2>` section should be before `<h2>SEE ALSO</h2>` section.